src/app/pages/maps/bubble/bubble-map.component.ts

```diff
@@ -2,8 +2,8 @@ import { Component, OnDestroy } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { combineLatest } from 'rxjs';
 import { takeWhile } from 'rxjs/operators';
-import { NgxEchartsService } from 'ngx-echarts';
```

Comment:
Do we still need 'ngx-echarts' module ?

Reply:
As I see it properly, we're still using `echarts` directive in the our code.